content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -63,17 +63,16 @@ If you experience any issues running these scripts, see the [troubleshooting gui
 
 ## Kubeflow installation
 
-1. Run the following commands to download the latest `kfctl.sh`
+1. Download a ```kfctl``` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/).
```

Comment:
If we're moving from `kfctl.sh` to the `kfctl` binary, then we should remove this comment at line 10:

```
Note: Amazon Web Services (AWS) is moving from `kfctl.sh` to a command line interface (CLI) which gives you more control over your configuration and better reliability.
```

And also update the mention of `kfctl.sh` on lines 8-9 and line 15.

Reply:
Also, this related doc needs updating to use the binary instead of the script:
https://deploy-preview-763--competent-brattain-de2d6d.netlify.com/docs/aws/deploy/existing-cluster/

Reply:
Thanks for the pointers Sarah, will take a good look at it tomorrow, and will make PRs accordingly. If there's anything else specific before the doc sprint that needs touching up, I'd love to do so!